MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -2557,6 +2571,8 @@ - (void)localEventDidChangeIdentifier:(NSNotification *)notif
     
     if (event && previousId)
     {
+        MXLogVerbose(@"[MXKRoomDataSource][%p] localEventDidChangeIdentifier from: %@ to: %@", self, previousId, event.eventId);
```

Comment:
We can have this log before the `if` in case the notif to track malformed notif.

Reply:
Moved